src/@orb.yml

```
@@ -1,15 +1,8 @@
 version: 2.1
 
 description: >
-  Sample orb description
-# What will your orb allow users to accomplish?
-# Descriptions should be short, simple, and informative.
+  Create automated Testlio runs in your CI pipeline with the Testlio CLI orb integration.
 
-# This information will be displayed in the orb registry and is not mandatory.
 display:
-  home_url: "https://www.example.com/docs"
-  source_url: "https://github.com/<organization>/<project-name>"
-
-# If your orb requires other orbs, you can import them like this. Otherwise remove the "orbs" stanza.
-# orbs:
-#  hello: circleci/hello-build@0.0.5
+  home_url: "https://help.testlio.com"
```

Looking a t what other Orbs have in their profiles:

- https://circleci.com/developer/orbs/orb/codecov/codecov
- https://circleci.com/developer/orbs/orb/cypress-io/cypress
- https://circleci.com/developer/orbs/orb/rainforest-qa/rainforest
- https://circleci.com/developer/orbs/orb/postman/newman

I would suggest to follow the same practice and share link to main company website.

```suggestion
  home_url: "https://testlio.com"
```